doc/index.md

```diff
@@ -1,24 +1,23 @@
 #
-![The Rust Logo](img/logo/logo-color.png)
+![The AdaptiveCpp Logo](img/logo/logo-color.png)
 
-Welcome to the documentation of the AdaptiveCpp !
+Welcome to the documentation of AdaptiveCpp!
 
 <div class="grid cards" markdown>
 
 -   :material-clock-fast:{ .lg .middle } __Installation__
 
     ---
 
-    Configure and install AdaptiveCpp
+    Configure and install AdaptiveCpp.
 
     [:octicons-arrow-right-24: Installation](./installing.md)
 
--   :fontawesome-solid-gears:{ .lg .middle } __Can run on any hardware__
+-   :fontawesome-solid-gears:{ .lg .middle } __Can run on most hardware__
```

Comment:
> If we write "most", then this raises the question where we don't run...

Apple GPUs and other unholy concoctions. 😉

But, fair enough. I suppose this would allow us to designate any vendor we do not explicitly target (and whose hardware cannot be targeted via a vendor-agnostic backend) as simply \"not a major vendor\".